src/yup.ts

```diff
@@ -1,13 +1,21 @@
-import { Resolver, transformToNestObject, FieldValues } from 'react-hook-form';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { Resolver, transformToNestObject } from 'react-hook-form';
 import Yup from 'yup';
 
+/**
+ * From 0.32.0, Yup add TypeScript support and `path` typing is optional that's why we have `@ts-expect-error`
```

Comment:
@bluebill1049 I'm not satisfied, if you've a better solution

Reply:
> FYI: `path`: a string, indicating where there error was thrown. `path` is empty at the root level.

what do you mean by that?

Reply:
Sorry, I haven't played with the latest Yup, probably lack of context.

Reply:
This message is from https://github.com/jquense/yup#validationerrorerrors-string--arraystring-value-any-path-string

Reply:
oh, `path` is empty, basically means the object key is actually the path?

Reply:
Yes, the new type for ValidationError is:

```ts
export default class ValidationError extends Error {
    value: any;
    path?: string;
    type?: string;
    errors: string[];
    params?: Params;
    inner: ValidationError[];
    static formatError(message: string | ((params: Params) => string) | unknown, params: Params): any;
    static isError(err: any): err is ValidationError;
    constructor(errorOrErrors: string | ValidationError | ValidationError[], value?: any, field?: string, type?: string);
}
```


As you can see, `path` & `type` could be `undefined`

Reply:
Hmm, interesting, let's go with the `ignore` for now 👍 it's a bit inconsistent for the rest of the schema libs.

Reply:
Ok, PR is ready to review :)